TileStache/Pixels.py

```diff
@@ -100,7 +102,7 @@ def apply_palette(image, palette, t_index):
         
         indexes.append(mapping[(r, g, b)])
 
-    output = Image.fromstring('P', image.size, ''.join(indexes))
+    output = Image.frombytes('P', image.size, ''.join(indexes))
```

Comment:
Okay, in that case, I will do a complete check and will replace all of them. thanks for pointing it out.